packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -178,15 +178,20 @@ class SystemRulesTable extends React.Component {
         cells: [{
             title: (
                 <React.Fragment key={ key }>
-                    <Stack id={ `rule-description-${key}` } gutter="md">
```

Comment:
This gutter feature seems broken. It puts spacing between components, but it's the same regardless of sm,md,lg that you pass. So I just did it manually on the `StackItem` bottom margin.

Reply:
:+1: